app/models/course/question_assessment.rb

```diff
@@ -29,7 +29,7 @@ def display_title(num = nil)
     return question_num if question.title.blank?
 
     I18n.t('activerecord.course/assessment/question.question_with_title',
-           question_number: question_num, title: question.title)
+           question_number: question_num)
```

Comment:
Have you considered possible regressions due to this change?

Reply:
ditto